pkg/apis/networking/register.go

```diff
@@ -33,37 +56,30 @@ const (
 	// annotation value for the Ingress it uses.  Based on such
 	// value a different reconciliation logic may be used (for examples,
 	// Istio-based Ingress will reconcile into a VirtualService).
-	IngressClassAnnotationKey = "networking.knative.dev/ingress.class"
+	IngressClassAnnotationKey = PublicGroupName + "/ingress-class"
```

Comment:
This annotation (`ingress.class`) was aligned with Kubernetes's ingres class annotation `kubernetes.io/ingress.class`.
But kubernetes started using `spec.ingressClassName` so we might be better to consider to use `spec` in the future.
https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation